unit_tests/test_cluster.py

```diff
@@ -223,6 +232,23 @@ def test_compaction_stopped_exception_is_catched(self):
             assert event_backtrace2["type"] == "COMPACTION_STOPPED"
             assert event_backtrace2["line_number"] == 1
 
+    def test_appending_to_log(self):
+        logs = """
+INFO  2022-07-14 09:28:34,095 [shard 1] database - Flushing non-system tables
+Reactor stalled for 32 ms on shard 1. Backtrace: 0x4e0d6e2 0x4e0c340 0x4e0d5f0 0x7f230af20a1f 0x14de339 0x14dc3b0 0x14d680c 0x14d5ff6 0x14e4d52 0x14d0238 0x1aec3c6 0x508c7e1
+kernel callstack:
+INFO  2022-07-14 09:28:35,102 [shard 1] database - Flushed non-system tables
+        """
+
+        self._read_and_publish_events(logs)
+
+        with self.get_raw_events_log().open() as events_file:
+            events = [json.loads(line) for line in events_file]
+            event = events[-1]
+            assert event["type"] == "REACTOR_STALLED"
+            assert event["line_number"] == 2
+            assert 'Reactor stalled for 32 ms on shard 1' in event['line']
```

Comment:
I'd suggest adding verification if we don't duplicate events by e.g. assertion there's only one REACTOR_STALLED event

Reply:
How that's related to fix in this PR ? i.e. you suggest testing we don't read the whole file again ?

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/5021/files#diff-624201d84c3151d182a7d818e70a3b002de54938a45ebe1a35365774ab7d77d8R160
`self._last_log_position` was changed - suggesting previously it was incorrect and unit test didn't find it.

Reply:
added this check